content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -133,507 +157,595 @@ types available by default:
-- Responsable Monotributo.
-- IVA liberado.
+- :guilabel:`B. Tax amount included`: this means that the taxed amount is included as part of the
+  product price, subtotal, and totals. This condition applies when the customer has the following
+  AFIP Responsibility types:
 
-.. image:: argentina/argentina15.png
-   :align: center
+  - IVA Sujeto Exento.
+  - Consumidor Final.
+  - Responsable Monotributo.
+  - IVA liberado.
 
+  .. image:: argentina/tax-amount-included.png
+     :align: center
+     :alt: Tax amount included.
 
 Special Use Cases
 ~~~~~~~~~~~~~~~~~
 
 Invoices for Services
 *********************
-For electronic invoices that include Services, the AFIP requires to report the  service starting
-and ending date, this information can be filled in the tab “Other Info”: 
 
-.. image:: argentina/argentina_edi_06.png
-   :align: center
+For electronic invoices that include :guilabel:`Services`, the AFIP requires to report the service
+starting and ending date, this information can be filled in the tab :guilabel:`Other Info`.
 
-If  the dates are not selected manually before the invoice is validated, the values will be
-filled automatically considering the beginning and day of the invoice month:
+.. image:: argentina/invoices-for-services.png
+   :align: center
+   :alt: Invoices for Services.
 
+If the dates are not selected manually before the invoice is validated, the values will be
+filled automatically with the first and last day of the invoice's month.
 
-.. image:: argentina/argentina_edi_07.png
+.. image:: argentina/service-dates.png
    :align: center
+   :alt: Service Dates.
 
 Exportation Invoices
 ********************
-The invoices related to Exportation transactions required a Journal that used the AFIP POS 
-System “Expo Voucher - Web Service” so the proper document type be associated:
 
-.. image:: argentina/argentina_edi_08.png
+Invoices related to :guilabel:`Exportation Transactions` require that a Journal use the AFIP
+POS System **Expo Voucher - Web Service** so that the proper document type(s) can be associated:
+
+.. image:: argentina/exporation-journal.png
    :align: center
+   :alt: Exporation journal.
 
-When the customer selected in the Invoice has set the AFIP responsibility type as “Cliente / 
-Proveedor del Exterior” or “IVA Liberado – Ley Nº 19.640”, Odoo automatically assigned:
+When the customer selected in the Invoice has set the AFIP responsibility type as
+:guilabel:`Cliente / Proveedor del Exterior` or :guilabel:`IVA Liberado - Ley Nº 19.640`, Odoo
+automatically assignes the:
 
-- Journal related to the exportation Web Service.
-- Exportation document type .
-- Fiscal position: Compras/Ventas al exterior.
-- Concepto AFIP:  Products / Definitive export of goods.
-- Exempt Taxes. 
+- Journal related to the exportation Web Service;
+- Exportation document type;
+- Fiscal position: Compras/Ventas al exterior;
+- Concepto AFIP: Products / Definitive export of goods; and
+- Exempt Taxes.
```

Comment:
Hi @vbe-odoo and @gza-odoo, just confirming that this section is 100% correct - thank you! 

![When the customer selected in the Invoice has set the AFIP responsibility](https://user-images.githubusercontent.com/36018073/191799293-a33efd87-0d1a-428a-81d6-047663b2e0c9.png)

Reply:
Quick follow up:
Please confirm the edits I made in [this comment](https://github.com/odoo/documentation/pull/2430#discussion_r977048191) above the screenshot (in green) are clear/correct, thank you 🙏 
@gza-odoo @vbe-odoo 

cc: @samueljlieber

Reply:
Hi, @samueljlieber so the functional description it's actually correct.
But probably I will just rephrase it a bit to make more clear:
"...When the customer selected in the Invoice is configured with an AFIP responsibility type **Cliente / Proveedor del Exterior - Ley N° 19.640**, Odoo automatically assigns the: ..."

But feel free to improve it if needed, functional wise it's correct :)

cc' @gza-odoo @StraubCreative

Reply:
Thanks @vbe-odoo we can make that adjustment.

@samueljlieber in addition to rewording suggestion from Valentino, can you also double check to see if the typo that says `assignes` is still present?

@vbe-odoo sanity check. Are these adjustments I made to the list below ok? 👇 
I removed the nested colons (`:`) since grammatically that's not correct, and replaced them with `as`, indicating that those are the assignments for each field.

```suggestion
- Journal related to the exportation Web Service;
- Exportation document type;
- Fiscal position as **Compras/Ventas al exterior**;
- Concepto AFIP as **Products / Definitive export of goods**; and
- Exempt Taxes.
```

Please let me know if this is not correct!

cc: @gza-odoo

Reply:
@StraubCreative thanks for the catch, yes those are great for me 💯 !
Thanks!
cc'  @samueljlieber @gza-odoo